helper/include/odp/helper/threads.h

```diff
@@ -118,6 +242,7 @@ int odph_odpthreads_create(odph_odpthread_t *thread_tbl,
  * (error occurs if any of the start_routine return non-zero or if
  *  the thread join/process wait itself failed -e.g. as the result of a kill)
  *
+ * @note Will be deprecated. Use odph_thread_join() instead.
```

Comment:
I think we should use `@deprecated`: http://www.doxygen.nl/manual/commands.html#cmddeprecated

Reply:
Fixed in v2. These helpers are documented now as deprecated, instead of "will be deprecated".